tests/graphql/passed/test-https-settings.graphql

```diff
@@ -0,0 +1,26 @@
+#> server-sdl
+schema @server(http: {
+  keepAliveInterval: 1,
+  keepAliveTimeout: 1
+  keepAliveWhileIdle: true,
+  poolIdleTimeout: 1024,
+  poolMaxIdlePerHost: 100
+}) {
+  query: Query
+}
+
+type User {
+    name: String
+    id: Int
+}
+
+type Query {
+    firstUser: User @http(path: "/users/1" baseURL: "http://jsonplaceholder.typicode.com")
+}
+
+#> client-query
+query @expect(json: {data:{firstUser:{name:"Leanne Graham"}}}) {
+    firstUser {
+        name
+    }
+}
```

Comment:
I don't think we need these tests.

Reply:
@tusharmath Are there any other ways to test?

Reply:
I think we can ignore the tests, as this is something that we can rely on for the compiler for. It is not possible to create an invalid client type.